packages/edit-site/src/components/global-styles/screen-css.js

```diff
@@ -54,6 +58,13 @@ function ScreenCSS() {
 					setEditorCanvasContainerView( undefined );
 				} }
 			/>
+			<Spacer margin={ 0 } paddingX={ 4 }>
+				<Navigator.Button as={ Button } variant="link" path="/blocks">
+					{ __(
+						'Want to change block appearance? Use the Blocкs section instead.'
```

Comment:
Haha, wrong "k" (I previously suggested a Cyrillic one by mistake)

```suggestion
						'Want to change block appearance? Use the Blocks section instead.'
```